tests/component/sections/dataset/dataset-files/files-table/files-info/file-info-cell/file-info-data/FileDate.spec.tsx

```diff
@@ -1,10 +1,16 @@
 import { FileDate } from '../../../../../../../../../src/sections/dataset/dataset-files/files-table/file-info/file-info-cell/file-info-data/FileDate'
 import { FileDateType } from '../../../../../../../../../src/files/domain/models/File'
+
 describe('FileDate', () => {
   it('renders the date', () => {
-    const date = { type: FileDateType.PUBLISHED, date: new Date('2023-09-18') }
+    const fileDate = new Date('2023-09-18')
+    const date = { type: FileDateType.PUBLISHED, date: fileDate }
     cy.customMount(<FileDate date={date} />)
-
-    cy.findByText(`Published Sep 18, 2023`).should('exist')
+    const dateString = fileDate.toLocaleDateString(Intl.DateTimeFormat().resolvedOptions().locale, {
+      year: 'numeric',
+      month: 'short',
+      day: 'numeric'
+    })
```

Comment:
Since this dateString is depending on the user's locale, this and the other changed tests are failing for me, because the FileDate component is implemented to always show the US date format and the test is checking the user's locale format, which in my case is the Spanish format

So either we change the test to check for the US date format or we refactor the FileDate component to show the user's date format.

In JSF I think it shows always the US format so that's why I implemented this like that, but I think that changing it to the user's locale could be an improvement, should we discuss this with Jullian?

Reply:
Ok, I see what you mean about locale format, that makes sense.  I think it's a good idea to use the user's  locale format.  I made the change because my local time zone was showing a different day since I'm five hours ahead, but that is fixed by testing against a dynamic date string, rather than changing the locale.
I can put a message in dv-tech to confirm about using the user's locale format, since it's different from the JSF